code/modules/mining/lavaland/weather_types.dm

```diff
@@ -88,8 +91,8 @@
 /datum/weather/ash_storm/false_alarm //No storm, just light ember fall
 	purely_aesthetic = TRUE
 	duration_overlay = "light_ash"
-	duration_message = "Looks like the storm just missed the mining area. False alarm."
-	wind_down_message = "The ash fall starts to trail off."
+	duration_message = "<span class='notice'>Gentle ashfall surrounds you like grotesque snow. The storm seems to have passed you by.</span>"
```

Comment:
Some of these descriptions seem a bit flowery for stuff you're gonna have in big red text every couple minutes